src/src_V2/modules/manager/account/index.jsx

```diff
@@ -0,0 +1,243 @@
+import { useEffect, useState } from 'react';
+import {
+  Box,
+  Grid,
+  Button,
+  Divider,
+  TextField,
+  IconButton,
+  Typography,
+} from '@mui/material';
+import { useFormik } from 'formik';
+import FileUploadIcon from '@mui/icons-material/FileUpload';
+import { useNavigate, useSearchParams } from 'react-router-dom';
+
+import CompanyInfo from './companyInfo';
+import ImgPlaceholder from './imgPlaceholder';
+import ChangePasswordDialog from './changePasswordDialog';
+import PageWrapper from '../../../components/userPageWrapper';
+import EditUserActions from '../../../components/editUserActions';
+
+import { ROUTES } from '../routes';
+import styles from './styles.module.css';
+import { getBase64 } from '../../../utils';
+import { accountMockData } from './mock/data';
+import {
+  accountIntialValues,
+  accountValidationSchema,
+} from '../../../utils/schemas/account/account';
+
+function AccountContent() {
+  const navigate = useNavigate();
+
+  const [initialData, setInitialData] = useState({});
+
+  const submitHandler = () => {
+    navigate('/manager/' + ROUTES.DASHBOARD);
+  };
+
+  const formikData = {
+    initialValues: accountIntialValues(initialData),
+    validationSchema: accountValidationSchema(),
+    onSubmit: submitHandler,
+    enableReinitialize: true,
+  };
+
+  useEffect(() => {
+    // TODO: Change later with async request
+    setInitialData(accountMockData);
+  }, []);
+
+  const { values, errors, touched, handleSubmit, getFieldProps, setFieldValue } =
+    useFormik(formikData);
+
+  const handleFileUpload = (e) => {
+    if (e.currentTarget.files[0]) {
+      getBase64(e.currentTarget.files[0]).then((res) => {
+        setFieldValue('companyLogo', res);
+      });
+    }
+  };
+
+  const [searchParams] = useSearchParams();
+
+  const isUser = Boolean(searchParams.get('isUser'));
+
+  // Change password dialog
+  const [open, setOpen] = useState(false);
```

Comment:
I think useToggle will work

Reply:
added useToggle hook, but for dialogs we force state to be true/false so user doesn't get unexpected results when double-clicking on toggle

Reply:
p.s. after merging with dev there was use-toggle hook so I deleted the one in this branch and used dev version